data/json/items/tools.json

```diff
@@ -8319,5 +8319,56 @@
     "symbol": ",",
     "color": "light_gray",
     "flags": [ "NO_SALVAGE" ]
-  }
+  },
+  {
+    "id": "autobread",
+    "type": "GENERIC",
+    "category": "food",
+    "name": "Libotech Autobread",
+    "description": "This is the Libotech Autobread battery powered bread machine. Originally marketed as a camping appliance, this machine makes coming home to a fresh loaf of bread a snap!",
+    "weight": 52,
+    "volume": 1,
+    "price": 10,
+    "bashing": 8,
+    "material": [ "metal", "glass" ],
+    "symbol": "%",
+    "color": "white"
+  },
+  {
+    "id": "autobread_process",
+    "type": "GENERIC",
+    "category": "food",
+    "name": "Libotech Autobread(Churning)",
```

Comment:
Space before bracket, and de-capitalize `churning`. The same for `(Done)` below.